sycl/test-e2e/Matrix/Inputs/joint_matrix_half_impl.hpp

```diff
@@ -116,33 +134,60 @@ int main() {
       continue;
 
     if (combinations[i].nsize == 0) { // Intel AMX
-      test<float, half, 2, /*TM*/ 16, /*TN*/ 16, /*TK*/ 32>();
+      test<half, float, float, 2, /*TM*/ 16, /*TN*/ 16, /*TK*/ 32>();
       break;
     }
 
     if (combinations[i].nsize == 16) { // architecture::intel_gpu_pvc
-      test<float, half, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
-      // test<half, half, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      test<half, float, float, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      test<half, half, half, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
```

Comment:
if this combination works with sg=32, then `all bfloat16` should also work for 8x16x16. Is not it the case?
For some reason similar combination is under sg=32 macro in previous tests (bfloat16 tests).